### ISSUE TITLE
Frequent travis mac errors

### ISSUE THREAD

**Issue by Noordfrees** [widelands/widelands#3796]
Many PR travis checks currently fail on mac for a variety of reasons:

- Timeout while linking:
    ~~~~
    [ 98%] Linking CXX executable test_economy

    No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

    Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

    The build has been terminated
    ~~~~
    e.g. https://travis-ci.org/github/widelands/widelands/jobs/665779085
- Strange other timeout:
    https://travis-ci.org/github/widelands/widelands/jobs/665777308
- Very very annoying:
    `An error occurred while generating the build script.` 
    (e.g. https://travis-ci.org/github/widelands/widelands/jobs/665142779)

Not sure if this is a problem of travis/github or of widelands, but this happens so often currently that I think it's worth a bug report

**Comment by stonerl** [comment]
I'll look into it.

**Comment by stonerl** [comment]
> 
>     * Strange other timeout:
>       https://travis-ci.org/github/widelands/widelands/jobs/665777308

Problem with the cache on the Travis' side.

>     * Very very annoying:
>       `An error occurred while generating the build script.`
>       (e.g. https://travis-ci.org/github/widelands/widelands/jobs/665142779)
> 

Also, a Travis problem.

It seems as if they might have problems with their mac-VMs.

**Comment by Noordfrees** [comment]
The errors in OP haven't happened again for some time, but now all mac builds are failing with
```
 CMake Error at /usr/local/Cellar/cmake/3.17.0_1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  Could NOT find Gettext (missing: GETTEXT_MSGMERGE_EXECUTABLE
  GETTEXT_MSGFMT_EXECUTABLE)
```
e.g. https://travis-ci.org/github/widelands/widelands/jobs/674454003

**Comment by Noordfrees** [comment]
That one was fixed by a1d170a6a8cfb41a53e5e339f4acd644351abad3
I guess we can consider this issue closed for now…
